frontend/src/app/features/grade-level-selection/grade-level-selection.component.ts

```diff
@@ -7,7 +7,7 @@ import config from '../../core/constants/config.json';
   // eslint-disable-next-line @angular-eslint/component-selector
   selector: 'grade-level-selection',
   templateUrl: './grade-level-selection.component.html',
-  styleUrls: ['./grade-level-selection.component.scss'],
+  styleUrls: ['./grade-level-selection.component.scss']
```

Comment:
There should be a comma here per ESLint.

Reply:
I added a comma.